packages/wallets/src/components/WalletsCard/WalletsCard.tsx

```diff
@@ -25,17 +26,19 @@ const WalletsCard: React.FC<React.PropsWithChildren<TProps>> = ({ children, rend
 
     return (
         <div
-            className={`wallets-card wallets-card ${isDemo ? 'wallets-card wallets-card--virtual' : ''}`}
```

Comment:
This was reusing the same styling `wallets-card` from `WalletCard`

Reply:
I believe we can keep this as `wallets-card` and rename the other one to `wallet-card` and update it's references. That way, it remains consistent with the component name.

Reply:
We have to prefix all classnames with `.wallets-` to avoid conflicts with stylings from other packages 😅

Reply:
<img width="1080" alt="Screenshot 2024-04-04 at 2 37 49 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/a3cc85ad-f4af-4515-ad86-34e04d89d57b">

Reply:
In that case, can we rename WalletsCard to WalletsContainer or something like that?